src/libs/actions/Session/index.js

```diff
@@ -115,6 +115,40 @@ function resendValidationLink(login = credentials.login) {
     API.write('RequestAccountValidationLink', {email: login}, {optimisticData, successData, failureData});
 }
 
+/**
+ * Request a new validate / magic code for user to sign in via passwordless flow
+ *
+ * @param {String} [login]
+ */
+function resendValidateCode(login = credentials.login) {
+    const optimisticData = [{
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
+        key: ONYXKEYS.ACCOUNT,
+        value: {
+            isLoading: true,
+            errors: null,
+            message: null,
+        },
+    }];
```

Comment:
Changing the `loading` state to `true` caused a "regression". This made the sign in button show a loader which is kinda odd because the user didn't click on that button. (Coming from https://github.com/Expensify/App/issues/22083)

Reply:
makes sense, thank you!